_develop/connect/get-started.md

```diff
@@ -0,0 +1,138 @@
+```
+
+## Usage
+
+### AuthOptions
+
+Every major method you'll use with `connect` requires you to pass some options, like the name and icon of your app, and what to do when authentication is finished. In practice, this means you need to define these options, and pass them to the various API methods.
+
+The exact interface you'll use [is defined as](https://github.com/blockstack/connect/blob/master/src/auth.ts#L12:L24):
+
+```typescript
+export interface AuthOptions {
+  // The URL you want the user to be redirected to after authentication.
+  redirectTo: string;
+  manifestPath?: string;
+  finished?: (payload: FinishedData) => void;
+  authOrigin?: string;
+  sendToSignIn?: boolean;
+  userSession?: UserSession;
+  appDetails: {
+    name: string;
+    icon: string;
+  };
+}
+```
+
+- `redirectTo`: The path in your app where users go after sign in.
+- `appDetails`: an optional object which includes `appName: string` and `appIcon: string`. This will speed up the process of loading your app's information during onboarding.
+- `manifestPath`: __(optional)__ - the path in your app where your manifest.json file can be found
+- `finished`: __(optional)__ - A callback that can be invoked after authentication. This prevents having to do a whole page refresh in a new tab. One argument is passed to this callback, which is an object with `userSession` included. If included, then the `redirectTo` path is ignored, and the user will be logged in automatically.
+- `authOrigin`: __(optional)__ - The URL you'd like to use for authentication. Only necessary for local development of the authenticator.
+- `sendToSignIn`: __(optional)__ - defaults to `false`. Whether the user should go straight to the 'sign in' flow.
+- `userSession`: __(optional)__ - pass a `UserSession` instance to use for authentication. If it's not passed, `@blockstack/connect` will create one for you.
+
+### In React Apps
+
+If you're using `connect` in a React app, then the best option is to include `connect`'s React infrastructure and hooks in your React app.
+
+First, setup the `Connect` provider at the "top-level" of your app - probably next to wherever you would put a Redux provider, for example.
+
+```javascript
+import { Connect } from '@blockstack/connect';
+
+const authOptions = {
+  redirectTo: '/',
+  finished: ({ userSession }) => {
+    console.log(userSession.loadUserData());
+  },
+  appDetails: {
+    name: 'My Cool App',
+    icon: 'https://example.com/icon.png',
+  },
+};
+
+const App = () => (
+  <Connect authOptions={authOptions}>
+    // the rest of your app's components
+  </Connect>
+)
+```
+
+Later, when you want to begin the onboarding process, use the `useConnect` hook to get `connect`'s `doOpenAuth` method.
+
+```javascript
+import { useConnect } from '@blockstack/connect';
+
+const SignInButton = () => {
+  const { doOpenAuth } = useConnect();
+
+  return (
+    <Button onClick={doOpenAuth}>
+      Sign In
+    </Button>
+  )
+}
+```
+
+### In ES6 (non-React) apps
```

Comment:
I'm a bit confused as to how this relates to sign in methods provided by http://blockstack.github.io/blockstack.js/index.html...

Do we want to sunset/replace auth in blockstack.js? @markmhx

Reply:
I don't think so - Connect uses those APIs. This is more of a 'wrapper' around Blockstack auth, with added UI and the ability to execute a callback after auth (instead of only allowing redirects).